include/fw_info.h

```diff
@@ -383,7 +383,7 @@ typedef bool (*fw_info_ext_api_provide_t)(const struct fw_info *fwinfo,
  *
  * @details Invalidation happens by setting the @c valid value to INVALID_VAL.
  *
- * @note This function needs to have CONFIG_NRF_NVMC enabled.
+ * @note This function needs to have CONFIG_NRFX_NVMC enabled.
```

Comment:
Shouldn't it be `@option{CONFIG_NRFX_NVMC}`?

Reply:
Yes, fixed.